git-checkout-interactive

```diff
@@ -33,5 +33,10 @@ printf "\n${GREEN}Enter the branch number to be checked out: "
 read userInput
 userInput=$((userInput-1))
 
+if [ $userInput -gt ${#arr[@]} ] || [ $userInput -eq ${#arr[@]} ] || [ $userInput -eq -1 ]; then
```

Comment:
1. You can combine `[ $userInput -gt ${#arr[@]} ] || [ $userInput -eq ${#arr[@]} ]` by `-ge`
2. Use `[$userInput -lt 0]` instead of `[ $userInput -eq -1]`. This would handle all negative inputs

Reply:
@kaunteya It's done.